_docs/installation/partitioning/encrypted.md

```diff
@@ -251,6 +251,25 @@ When doing root on ZFS, LUKS does not influence the `root=`. You just have to
 specify something like `root=ZFS=mypool/root/whatever` and the initramfs will
 take care of the rest, provided the `crypttab` mappings are correctly set up.
 
+As `cryptsetup` currently does not detect the root filesystem when it's on ZFS
+(resulting in warnings while creating the initramfs which can be ignored),
+you need to use the `initramfs` option in `crypttab`, e.g.:
+
+```
+# echo crypt /dev/sda3 none luks,initramfs > /etc/crypttab
+```
+
+If your ZFS pool consists of multiple encrypted devices, you can use a
```

Comment:
i'm not sure this part is something that belongs here; this is a more generic information about keyscripts

Reply:
I agree it's a bit offtopic but according to the man it's at least kind of specific to non-systemd systems. There are a few other snippets not chimera specific (e.g. `discards` option) but I think it makes sense to have them in the docs because they're so common, even if referenced in a minimal way.

The need to unlock multiple devices was a big issue for me when starting with ZFS (coming from raid1 with mdadm) and it took me waaaay to long to discover keyscripts (especially this ready-to-use solution), so I tried to add a simple/concise example. If you want that part removed, maybe I could replace it with just some reference, something like `to unlock multiple devices at boot, see ..`?

Reply:
the debian cryptsetup initramfs integration docs are relevant: https://cryptsetup-team.pages.debian.net/cryptsetup/README.initramfs.html

Reply:
maybe the docs should refer to them (though more generally, not specifically for this zfs case)